app.js

```diff
@@ -1,13 +1,25 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
 const logger = require('morgan');
+const rateLimiter = require('express-rate-limit');
 const { errorHandler } = require('./utils/errorHandler');
 const cors = require('cors');
 require('express-async-errors');
 
 const indexRouter = require('./routes/index');
 
+const limiter = rateLimiter({
```

Comment:
Moving this code block to a separate module will help, at least regarding reusability.